reference/forms/types/button.rst

```diff
@@ -51,3 +53,83 @@ as a key. This can be useful when you need to set a custom class for the button:
 .. include:: /reference/forms/types/options/button_label.rst.inc
 
 .. include:: /reference/forms/types/options/button_translation_domain.rst.inc
+
+label_translation_parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``array`` **default**: ``array()``
```

Comment:
We are now using `[]` for arrays. This should be changed everywhere in this pull request.